compiler/src/iree/compiler/Pipelines/Pipelines.cpp

```diff
@@ -129,9 +186,14 @@ void buildIREEPrecompileTransformPassPipeline(
     break;
   default:
     if (compileFrom < IREEVMPipelinePhase::Preprocessing) { // late-entry.
-      // Not a large enough phase for IREE_TRACE_ADD_[BEGIN,END]_FRAME_PASS.
+      IREE_TRACE_ADD_BEGIN_FRAME_PASS(passManager, "Preprocessing");
```

Comment:
Likely fine to add this back, but the Tracy UI was looking strange with how many frames we defined (lots of rows stacked vertically).

I had this screenshot on https://github.com/openxla/iree/pull/15216:
![image](https://github.com/openxla/iree/assets/4010439/1217edcf-4e3a-4f32-adf1-799012728565)

Reply:
it's just one more line, and if someone is using preprocessing it's useful to see - would be better to fix tracy :P